config/robots.rb

```diff
@@ -4276,4 +4276,347 @@
+          states: ['pending'],
+          label: 'Bed 3',
+          parent: bed(4).barcode,
+          target_state: 'passed'
+        },
+        bed(9).barcode => {
+          purpose: 'RVIG Lig',
+          states: ['passed'],
+          label: 'Bed 9'
+        },
+        bed(8).barcode => {
+          purpose: 'RVIG Lig Bind',
+          states: ['pending'],
+          label: 'Bed 8',
+          parent: bed(9).barcode,
+          target_state: 'passed'
+        }
+      }
+    )
+
+    custom_robot(
+      'bravo-rvig-lig-bind-to-rvig-lib-pcr',
+      name: 'Bravo RVIG Lig Bind => RVIG Lib PCR',
+      beds: {
+        bed(5).barcode => {
+          purpose: 'RVIG Lig Bind',
+          states: ['passed'],
+          label: 'Bed 5'
+        },
+        bed(13).barcode => {
+          purpose: 'RVIG Lib PCR',
+          states: ['pending'],
+          label: 'Bed 13',
+          parent: bed(5).barcode,
+          target_state: 'passed'
+        }
+      }
+    )
+
+    custom_robot(
+      'hamilton-rvig-lib-pcr-to-rvig-lib-pcr-xp',
+      name: 'Hamilton RVIG Lib PCR => RVIG Lib PCR XP',
+      beds: {
+        bed(8).barcode => {
+          purpose: 'RVIG Lib PCR',
+          states: ['passed'],
+          label: 'Bed 8'
+        },
+        bed(10).barcode => {
+          purpose: 'RVIG Lib PCR XP',
+          states: ['pending'],
+          label: 'Bed 10',
+          parent: bed(8).barcode,
+          target_state: 'passed'
+        },
+        bed(13).barcode => {
+          purpose: 'RVIG Lib PCR',
+          states: ['passed'],
+          label: 'Bed 13'
+        },
+        bed(15).barcode => {
+          purpose: 'RVIG Lib PCR XP',
+          states: ['pending'],
+          label: 'Bed 15',
+          parent: bed(13).barcode,
+          target_state: 'passed'
+        }
+      }
+    )
+
+    custom_robot(
+      'beckman-rvig-lib-pcr-xp-to-rvig-lib-prepool',
+      name: 'Beckman RVIG Lib PCR XP => RVIG Lib PrePool',
+      beds: {
+        bed(5).barcode => {
+          purpose: 'RVIG Lib PCR XP',
+          states: %w[passed],
+          child: bed(9).barcode,
+          label: 'Bed 5'
+        },
+        bed(6).barcode => {
+          purpose: 'RVIG Lib PCR XP',
+          states: %w[passed],
+          child: bed(9).barcode,
+          label: 'Bed 6'
+        },
+        bed(7).barcode => {
+          purpose: 'RVIG Lib PCR XP',
+          states: %w[passed],
+          child: bed(9).barcode,
+          label: 'Bed 7'
+        },
+        bed(8).barcode => {
+          purpose: 'RVIG Lib PCR XP',
+          states: %w[passed],
+          child: bed(9).barcode,
+          label: 'Bed 8'
+        },
+        bed(9).barcode => {
+          purpose: 'RVIG Lib PrePool',
+          states: %w[pending],
+          parents: [bed(5).barcode, bed(6).barcode, bed(7).barcode, bed(8).barcode],
+          target_state: 'passed',
+          label: 'Bed 9'
+        }
+      },
+      destination_bed: bed(9).barcode,
```

Comment:
odd that PoolingRobot needs that destination_bed defined specifically in addition to the bed being the only one having parents above.

Reply:
Yes I thought that too, maybe this config changed over time and this became redundant / removable.